src/test/java/com/kaltura/client/test/tests/servicesTests/bookmarkTests/AddTests.java

```diff
@@ -23,46 +23,57 @@
     private int fileId;
     private BookmarkActionType actionType;
     private int position = 0;
+    // instantiate Bookmark object
+    private Bookmark bookmark = new Bookmark();
+    // instantiate BookmarkPlayerData object
+    private BookmarkPlayerData playerData = new BookmarkPlayerData();
+    // instantiate BookmarkFilter object
+    private BookmarkFilter bookmarkFilter = new BookmarkFilter();
 
     @BeforeClass
     private void add_tests_before_class() {
+        // Get asset id from ingest
         assetId = mediaAsset.getId();
-
         type = AssetType.get("MEDIA");
         AssetReferenceType assetReferenceType = AssetReferenceType.get("MEDIA");
         Response<Asset> assetResponse = AssetServiceImpl.get(sharedMasterUserKs,String.valueOf(assetId),assetReferenceType);
         fileId = assetResponse.results.getMediaFiles().get(0).getId();
         actionType = BookmarkActionType.get("FIRST_PLAY");
-    }
-
-    @Description("bookmark/action/add - basic functionality")
-    @Test
-    private void add() {
 
-        Bookmark bookmark = new Bookmark();
+        // Initialize bookmark object parameters
         bookmark.setType(type);
         bookmark.setId(String.valueOf(assetId));
         bookmark.setPosition(position);
 
-        BookmarkPlayerData playerData = new BookmarkPlayerData();
+        // Initialize playerData object parameters
         playerData.setAction(actionType);
         playerData.setAverageBitrate(0);
         playerData.setTotalBitrate(0);
         playerData.setCurrentBitrate(0);
-
         playerData.setFileId((long) fileId);
-
         bookmark.setPlayerData(playerData);
 
+        // Initialize asset id
+        bookmarkFilter.setAssetIdIn(String.valueOf(assetId));
+        // Initialize orderBy
+        bookmarkFilter.setOrderBy("POSITION_ASC");
```

Comment:
Please use BookmarkOrderBy.POSITION_ASC and avoid of using String constants everywhere instead of ENUMs